src/app/pages/homepage/homepage.component.ts

```diff
@@ -1,15 +1,16 @@
 import { Component, OnInit } from '@angular/core';
 import {AuthService} from '../../../services/authservice';
-import {Router} from '@angular/router';
+import {MessageService} from '../../../services/messageservice';
 
 @Component({
   selector: 'app-homepage',
   templateUrl: './homepage.component.html',
   styleUrls: ['./homepage.component.css']
 })
 export class HomepageComponent implements OnInit {
-  public chat: any = { message: '' };
-  public chatMessages: any[] = [];
+  public userEmail: '';
+  public adminUserId: boolean;
+  public newsInputOpen: boolean;
```

Comment:
Apply some value to statment on declaration. Example: public newsInputOpen: boolean = false; In this case you will be able to manipulate with this value in your functions.